src/commands.jl

```diff
@@ -0,0 +1,88 @@
+# TS.CREATE key [RETENTION retentionTime] [UNCOMPRESSED] [LABELS field value..]
+function create(conn::RedisConnection, key::String, command::String)
+	response = execute_command(conn, ["TS.CREATE", key, command])
+	println(response)
+end
+
+# TS.ALTER key [RETENTION retentionTime] [LABELS field value..]
+function alter(conn::RedisConnection, key::String, params::String)
+	response = execute_command(conn, ["TS.ALTER", key, params])
+	println(response)
+end
+
+# TS.ADD call with specified key-value pairs for LABELS
+function add(conn::RedisConnection, key::String, timestamp::Int64, value::Float64, labels::Dict{String,String}, retention=0)
+	# TS.ADD key timestamp value [RETENTION retentionTime] [UNCOMPRESSED] [LABELS field value..]
+	# Note that UNCOMPRESSED is unused in timeseries-py; we are following suit here.
+	label_str = ""
+	for i in keys(labels) label_str = string(label_str, "$i $(labels[i]) ") end
+	response = execute_command(conn, ["TS.ADD", key, string(timestamp), string(value), retention, label_str])
+	println(response)
+end
+
+# TS.ADD without labels
+function add(conn::RedisConnection, key::String, timestamp::Int64, value::Float64, retention=0)
+	# TS.ADD key timestamp value [RETENTION retentionTime] [UNCOMPRESSED] [LABELS field value..]
+	# Note that UNCOMPRESSED is unused in timeseries-py; we are following suit here.
+	response = execute_command(conn, ["TS.ADD", key, string(timestamp), string(value), retention])
+	println(response)
```

Comment:
@jeffreylovitz I think it's better to remove the println and return the timestamp